src/lwc/geConstants/geConstants.js

```diff
@@ -9,9 +9,21 @@ const HTTP_CODES = Object.freeze({
     Request_Timeout: 408,
 });
 
+const PAYMENT_TRANSACTION_STATUS_ENUM = Object.freeze({
```

Comment:
Before I review, please take a look at https://github.com/SalesforceFoundation/NPSP/pull/5395/files#diff-a2c9f4a7190565f47b4b3e2b3a6a9e31. We have decided to move the transaction status enum in apex in Daniel's PR.